docs/changes/cosmology/12693.api.rst

```diff
@@ -0,0 +1,2 @@
+Lowercase ``m_nu`` initialization error message and add information to the
```

Comment:
I'm still not sure about change-log entries like these. If I look at the API change section, I'd like not to be distracted from minimal ones that do not influence usage...  But up to you...

Reply:
I agree. I don't think it's 100 codified what are our guidelines for these small API changes. I've tended to err on the side of caution. But I'm also happy to not make the new file and squash the commit.